CONTRIBUTING.rst

```diff
@@ -152,6 +152,11 @@ you can use the following command to re-serialize the whole repo::
 
     $ python tools/reserialize.py --all .
 
+**Note:** Before using the tools, you should install some packages. In order to
```

Comment:
You can use [a `.. note::` directive](http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions). I'm not sure how it renders on GitHub though.